### DISCUSSION TITLE
how to remove server name from response header

### DISCUSSION THREAD

**Discussion by SeongKwan** [vercel/community#813] [Help] [answered]
Currently, nextjs-based web service is being operated through vercel.

The server name displayed in the response header must be hidden due to security policy.

I would really appreciate it if you could tell me how.

![image](https://user-images.githubusercontent.com/34581795/187651974-50103e3a-e812-42cd-b896-8820201129f1.png)

**Comment by swarnava** [comment, marked as answer]
Hi,

Unfortunately this cannot be disabled and this isn't only Next.js specific but in any app.  Also, it will be quite obvious that you are using Vercel anyhow, based on the IP addresses.

**Reply by SeongKwan** [reply]
thank you very much for your answer.
